components/utility/PerformanceOptimizer.tsx

```
@@ -0,0 +1,78 @@
+      {/* DNS prefetch for external domains */}
+      <link rel="dns-prefetch" href="//www.fussball.de" />
+      <link rel="dns-prefetch" href="//msv61.fan12.de" />
+      
+      {/* Preconnect to critical external resources */}
+      <link rel="preconnect" href="https://www.fussball.de" crossOrigin="anonymous" />
+      
+      {/* Resource hints for better LCP */}
+      <link rel="preload" as="image" href="/logo.svg" />
+      
+      {/* Critical CSS for above-the-fold content */}
+      <style dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)